### ISSUE TITLE
Density for bounded distributions

### ISSUE THREAD

**Issue by drcthal** [ignacio82/vizdraws#36]
This is more of a general kdensity issue, but if you feed in a bounded distribution, the distribution plot will show/color area outside of the bounds, while the bar plot will (correctly) not reflect that.
e.g. 
set.seed(123)
prior <- runif(100)
IMPosterior(prior = prior)

**Comment by ignacio82** [comment]
Good catch. The only solution I can think is adding an argument "bounded" that you can set to `TRUE` and if you do that we can avoid plotting things outside the domain of the draws that are passed. Do you think that would work? do you have a better idea?

**Comment by ignacio82** [comment]
On seconds thoughts maybe the argument that we want to add is `support` and just pass that to `kdensity`

**Comment by drcthal** [comment]
Hmm, functionally it looks like calculating densities with kdensity and support doesn't look too different from just calculating regular densities and chopping them off, so for simplicity (not changing the code and adding the dependency) I think sticking with density and chopping the ends off makes sense. It'd still be good to call the option support for consistency.

**Comment by ignacio82** [comment]
ok
